spec/std/string_spec.cr

```diff
@@ -2578,3 +2599,9 @@ describe "String" do
     end
   end
 end
+
+private def with_io_memory
```

Comment:
Why did you not just use `String.build` instead of this helper?

Reply:
Oooh... it's a long time since I programmed in Crystal, I forgot this already 😅

Reply:
It's my bad, I didn't check the specs well enough!